### ISSUE TITLE
Final Peer Review #zz274

### ISSUE THREAD

**Issue by EricZhouZhuo** [muyuanliu1/Predicting-Accident-Severity#11]
Things I like:
1. The team made a very detailed exploration of data pre-processing by identifying and imputing missing values, data encoding, resampling.
2. The modeling part is well-organized by starting with logistic regression and improving with random forest. Finally, they used control burn to make a feature selection, resulting in a more interpretable model. They generated good and confident results by considering overfitting and hyperparameter tuning. 
3. The applications and business cases that the model could be used impressed me. They identified the 5 most important factors and demonstrated them thoroughly on different use cases. 

Things that could be improved:
1. Thing the outcome variable is ordinal, we could also check for the correlation matrix between the independent and dependent variables, and try some regression approaches. Regression models are usually easier for us to interpret than tree-based models.
2. I kindly think that accuracy is not the most important objective of this project since the team aimed to identify what are the important factors for the severity of car accidents. They could somehow include more features into the model. However, again, this is one of the disadvantages of the tree-based model. They could try some other classification models.
3. To make the model more applicable to real cases, I think the area to be considered can be outside California. For the direction of future works, they could find similar datasets for other states in the US, and before fitting the model, they might need to cluster similar states into the same group and fit different models on different clusters.
